SETUP.md

```diff
@@ -0,0 +1,74 @@
+###
+1. Clone the repository to your machine.
+```
+git clone https://github.com/boulder-food-rescue/food-rescue-robot.git
+cd food-rescue-robot
+bundle install
+```
+
+2. Copy `database.yml.dist`
+```
+cp config/database.yml.dist config/database.yml
+```
+
+3. Setup `config/database.yml` to look like:
+
+```
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  adapter: postgresql
+  database: bfr_webapp_db_test
+  host: localhost
+  pool: 5
+  timeout: 5000
+
+development:
+  adapter: postgresql
+  database: bfr_webapp_db
+  host: localhost
+  pool: 5
+  timeout: 5000
+
+production:
+  adapter: postgresql
+  database: bfr_webapp_db
+  username: bfr_webapp_user
+  password: CHANGEME
+  host: localhost
+  pool: 5
+  timeout: 5000
+```
+
+
+5. Prepare postgres
+```
+initdb bfr-data/
+```
+
+  * If you have an existing database by chance,
+```
+ps aux | grep postgres
+kill -9 (number from below)
```

Comment:
@Fosome @tmikeschu  is it typical to have instructions for killing an existing database in a README? I'm under the impression that most READMEs assume that you're working from a clean slate in that regard.